lib/get-outputs.js

```diff
@@ -1,6 +1,7 @@
 'use strict';
 
 const getCwd = require('./get-cwd');
+const runInterpolation = require('./run-interpolation');
```

Comment:
nit: `run` to me means a command, whereas I would just call this `interpolate()`